pkg/api/v1/cloud_provider_integration.go

```diff
@@ -0,0 +1,20 @@
+package v1
+
+// CloudProviderIntegration define an integration to a cloud provider
+type CloudProviderIntegration struct {
+	// ProviderName is the name of the cloud provider. Currently only AWS is supported.
+	ProviderName string `json:"providerName"`
```

Comment:
no validation annotations necessary here?

Reply:
It's a copy & paste from the existing definition, I preferred not to enforce new rules.

Reply:
@helderjs can you please submit a ticket so we don't forget to think about this field?